filebeat/tests/system/test_modules.py

```diff
@@ -178,6 +178,7 @@ def _test_expected_events(self, test_file, objects):
 
         for ev in expected:
             found = False
+            clean_keys(ev)
```

Comment:
This shouldn't be needed after #9506 if expected json is regenerated, I have opened #9862 to regenerate the expected json files and remove this line.